### PR TITLE
ref: Use es5 syntax for JS examples

### PR DESCRIPTION
**Pull request by HazAT**
To improve comaptabilty

### CODE REVIEW

File: src/collections/_documentation/platforms/node/koa.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ const Sentry = require('@sentry/node');
 Sentry.init({ dsn: '___PUBLIC_DSN___' });
 
 app.on('error', (err, ctx) => {
-  Sentry.withScope(scope => {
+  Sentry.withScope(function(scope) {
     scope.addEventProcessor(event => Sentry.Handlers.parseRequest(event, ctx.request));
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/koa.md]: The line below is also es6

File: src/collections/_documentation/platforms/node/index.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ import * as Sentry from '@sentry/node';
 
 Sentry.init({
   dsn: '___PUBLIC_DSN___',
-  integrations: integrations => {
+  integrations: function(integrations) {
     // integrations will be all default integrations
     return [...integrations, new MyCustomIntegration()];
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/index.md]: `...` is es6

File: src/collections/_documentation/platforms/javascript/react.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ class ExampleBoundary extends Component {
     }
 
     componentDidCatch(error, errorInfo) {
-      Sentry.withScope(scope => {
+      Sentry.withScope(function(scope) {
           scope.setExtras(errorInfo);
           const eventId = Sentry.captureException(error);
           this.setState({eventId});
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/javascript/react.md]: object shorthand notation is also `es6`

File: src/collections/_documentation/platforms/javascript/index.md [hunk 4]
```diff
@@ -1014,9 +1014,9 @@ import * as Sentry from '@sentry/browser';
 
 Sentry.init({
   dsn: '___PUBLIC_DSN___',
-  integrations: integrations => {
+  integrations: function(integrations) {
     // integrations will be all default integrations
-    return [...integrations, new MyCustomIntegration()];
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/javascript/index.md]: `return integrations.concat(new MyCustomIntegrations());`

File: src/collections/_documentation/platforms/javascript/index.md [hunk 5]
```diff
@@ -1001,7 +1001,7 @@ import * as Sentry from '@sentry/browser';
 
 Sentry.init({
   dsn: '___PUBLIC_DSN___',
-  integrations: integrations => {
+  integrations: function(integrations) {
     // integrations will be all default integrations
     return integrations.filter(integration => integration.name !== 'Breadcrumbs');
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/javascript/index.md]: `=>`

File: src/collections/_documentation/platforms/javascript/react.md [hunk 6]
```diff
@@ -50,10 +50,10 @@ class ExampleBoundary extends Component {
     }
 
     componentDidCatch(error, errorInfo) {
```
Comment (ypresto) [inline src/collections/_documentation/platforms/javascript/react.md]: class is es6, but babel or typescript is required to use React, so using es6 for react code is fine.

File: src/collections/_documentation/platforms/javascript/react.md [hunk 7]
```diff
@@ -50,10 +50,10 @@ class ExampleBoundary extends Component {
...
-      Sentry.withScope(scope => {
+      Sentry.withScope(function(scope) {
           scope.setExtras(errorInfo);
           const eventId = Sentry.captureException(error);
-          this.setState({eventId});
+          this.setState({eventId: eventId});
```
Comment (ypresto) [inline src/collections/_documentation/platforms/javascript/react.md]: `this` is something not a react component here, because arrow function is not used.

**Comment by kamilogorek** [comment]
Good point. Fixed in https://github.com/getsentry/sentry-docs/pull/1255 Thanks @ypresto

**Pull request merged**